src/mercury_engine_data_structures/construct_extensions/alignment.py

```diff
@@ -112,13 +112,14 @@ def PrefixedAllowZeroLen(lengthfield, subcon, includelengthfield=False):
     return FocusedSeq(
         "prefixed",
```

Comment:
I think that we should choose either the / format or the = format and standardize it across the repo. I'm partial to the = format because it feels more readable to me.

Reply:
This is probably more of a "we should pick one and manually update code to comply/enforce it on new PRs" than something that ruff can handle

Reply:
I'm fine having both.
I prefer the kwargs one, but it is more limiting, particularly having unnamed padding